lib/compat/wordpress-6.4/fonts/fonts.php

```diff
@@ -34,20 +34,20 @@ function wp_print_font_faces( $fonts = array() ) {
 		$wp_font_face = new WP_Font_Face();
 		$wp_font_face->generate_and_print( $fonts );
 	}
+}
 
-	// @core-merge: do not merge this code into Core.
-	add_filter(
-		'block_editor_settings_all',
-		static function( $settings ) {
-			ob_start();
-			// @core-merge: add only this line into Core's `_wp_get_iframed_editor_assets()` function after `wp_print_styles()`.
-			wp_print_font_faces();
-			$styles = ob_get_clean();
+// @core-merge: do not merge this code into Core.
+add_filter(
+	'block_editor_settings_all',
+	static function( $settings ) {
+		ob_start();
+		// @core-merge: add only this line into Core's `_wp_get_iframed_editor_assets()` function after `wp_print_styles()`.
```

Comment:
```suggestion
```
Should this comment be removed, since `wp_print_font_faces()` is already in core as of commit [c29b095](https://github.com/ironprogrammer/wordpress-develop/commit/c29b095e762836e47df9080d670c2f6570d6cd12#diff-edfc853e670c7bfc5a838dc74f75ba5d8b95cb4adbddf0644d9227bcb692e344R364)?

Reply:
cc @hellofromtonya

Reply:
>Should this comment be removed, since `wp_print_font_faces()` is already in core as of commit [c29b095](https://github.com/ironprogrammer/wordpress-develop/commit/c29b095e762836e47df9080d670c2f6570d6cd12#diff-edfc853e670c7bfc5a838dc74f75ba5d8b95cb4adbddf0644d9227bcb692e344R364)?

It is. However, the intent is to retain these merge notes for history as the compat files relate directly to the WP version.